test/multibuild/build.spec.ts

```diff
@@ -521,7 +521,7 @@ describe('Specifying a dockerfile', () => {
 				.to.have.property('error')
 				.that.has.property('message')
 				.that.matches(
-					/Dockerfile parse error line 1: unknown instruction: CORRECT/i,
+					/Dockerfile parse error on line 1: unknown instruction: CORRECT/i,
```

Comment:
This looks like an upstream change.
My best guess so far is that this comes directly from moby and we bumped docker on our CI or something.
See: https://github.com/moby/buildkit/blob/8da36b5c5cb0137addfe5a39fb0acc63eab17257/frontend/dockerfile/instructions/parse.go#L142

Reply:
Can we loosen the match a bit to support both variations? I would rather this test not depend heavily on which runners are used.